src/main/java/com/synopsys/integration/detect/lifecycle/run/operation/OperationRunner.java

```diff
@@ -186,6 +186,9 @@
 import com.synopsys.integration.util.OperatingSystemType;
 
 import com.synopsys.integration.blackduck.bdio2.util.Bdio2ContentExtractor;
+
+import net.minidev.json.JSONObject;
```

Comment:
I haven't compared functionality but is there any reason we needed this over something like Gson's JSONObject? If we could use Gson's version of this it might be preferable as that library is heavily used throughout Detect.

Reply:
Agreed, no special reason to use net.minidev. I think my IDE prioritized importing this one. I've changed it to Gson to avoid a new dependency, thanks!